pkg/tfbridge/detailed_diff.go

```diff
@@ -313,22 +313,20 @@ func (differ detailedDiffer) calculateSetHashIndexMap(
 		return nil
 	}
 
-	convertedVal, err := makeSingleTerraformInput(
-		differ.ctx, path.String(), resource.NewArrayProperty(listVal), tfs, ps)
-	if err != nil {
-		return nil
-	}
+	convertedElements := []interface{}{}
 
-	if convertedVal == nil {
-		return nil
+	for _, elem := range listVal {
```

Comment:
Narrate what's happening here? Is this equivalent to leaving makeSingleTerraformInput as above but passing the flag for it to generate *schema.Set values, if that was possible? Or something else is changing?

Reply:
Ah I see, this is good. Perhaps don't call the arg to this function `listVal` but `setElements`